guide/07-working-with-bigdata/03-performing_big_data_analysis.ipynb

```diff
@@ -0,0 +1,956 @@
+{
```

Comment:
<p>In the previous guide, you learnt how to register big data file share with your ArcGIS GeoAnalytics Server. Adding a big data file share to the Geoanalytics server adds a corresponding <a href="https://enterprise.arcgis.com/en/portal/latest/use/what-is-a-big-data-file-share.htm" target="_blank">big data file share item</a> in the portal. We can search for these types of items using the <code>item_type</code> parameter. </p><p>When you add a big data file share, a corresponding item gets created in your portal. You can search for it like any other portal <code><a href="https://developers.arcgis.com/rest/services-reference/item.htm" target="_blank">Item</a></code> and query its layers.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1076/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>